### DISCUSSION TITLE
v0.7.3

### DISCUSSION THREAD

**Discussion by KiyanYang** [KiyanYang/DotVast.HashTool.WinUI#45] [Announcements]
### 系统要求

最低 Windows 10 2004 (10.0.19041.0)

### 变更

- 移除 Keccak288 的支持
- 优化 QuickXor, SHA2, BLAKE2, KECCAK（但在某些 CPU 上可能出现性能下降）
- 修复主页计算模式下拉框的提示未本地化
- 一些依赖项更新

### 校验

|                  文件                  |                               SHA256                               |
| :------------------------------------: | :----------------------------------------------------------------: |
| DotVast.HashTool.WinUI-0.7.3-arm64.zip | `57E051D477683EDF5AD08C3C48EEAC23ACFED09A79E74DE86248BB572C6F87D5` |
|  DotVast.HashTool.WinUI-0.7.3-x64.zip  | `7527D0E292CB857512D2C29C4C114FA7339A098E1FCAADC1A838F191FB0268B7` |

<hr /><em>This discussion was created from the release <a href='https://github.com/KiyanYang/DotVast.HashTool.WinUI/releases/tag/v0.7.3'>v0.7.3</a>.</em>

**Comment by KiyanYang** [comment]
设置里读取的上下文菜单好像有问题

**Comment by rubenalamina** [comment]
I'm trying to install it but when I run the install.ps1 script nothing happens. Same if I run the Add-AppDevPackage.ps1 instead. Do I need to have developer mode enabled or any other requirements?

**Reply by KiyanYang** [reply]
Yes. Open `Settings`, Go to `System > For developers`, Activate `Developer Mode` and `PowerShell`.

![image](https://github.com/user-attachments/assets/0b454383-9213-481f-a5ab-d39c16b41bfa)
![image](https://github.com/user-attachments/assets/33f1d5d9-aec7-4f1d-a350-16faefc6ffd1)
